tutorials/experiments/llama-index/answer_and_context_relevancy_llamatrace.ipynb

```diff
@@ -0,0 +1,3268 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    if not os.getenv("PHOENIX-API-KEY"):
</pre><p>Maybe a screenshot or how to get this in the code comment could help</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>